catalog/spatial/registry/registry-common/src/test/java/org/codice/ddf/registry/common/metacard/TestRegistryObjectMetacardType.java

```diff
@@ -61,7 +48,7 @@ public void testConstruction() {
         assertThat(descriptors, not(nullValue()));
         assertThat(CollectionUtils.isEmpty(descriptors), is(false));
 
-        assertThat(registryObjectMetacardType.getAttributeDescriptor(Metacard.ID)
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(RegistryObjectMetacardType.REGISTRY_ID)
```

Comment:
We had previously been adding the common metacard attributes manually, but this has since changed. As such the RegistryObjectMetacardType no longer has the Metacard ID as an attribute, however registry metacards still have the core id attribute.